### DISCUSSION TITLE
Can we contruct neural networks arbitrarily within DeepXDE?

### DISCUSSION THREAD

**Discussion by reactingflow** [lululxvi/deepxde#1764] [Q&A]
Hello everyone!

I'm new to DeepXDE, and I'm wondering if it's possible to contruct neural networks arbitrarily within DeepXDE, like the one below. I've been browsing through the forum for related posts, but I haven't found a definitive answer yet. Can anyone answer my question? If it's feasible, please tell me how to implement it. Thank you!

![image](https://github.com/lululxvi/deepxde/assets/151173251/e516dbb6-5e2d-49a6-8466-fd18c0dfbe7f)

**Comment by lululxvi** [comment]
Not sure what exactly you want to do. But seems doable.
